appdatabase/database.go

```diff
@@ -76,6 +87,54 @@ func InitializeDB(path, password string, kdfIterationsNumber int) (*sql.DB, erro
 	return db, nil
 }
 
+func FixMissingKeyUIDForAccounts(sqlTx *sql.Tx) error {
+	rows, err := sqlTx.Query(`SELECT address,pubkey FROM accounts WHERE pubkey IS NOT NULL AND type != '' AND type != 'generated'`)
+	if err != nil {
+		log.Error("Migrating accounts: failed to query accounts", "err", err.Error())
+		return err
+	}
+	defer rows.Close()
+	for rows.Next() {
+		var address e_types.Address
+		var pubkey e_types.HexBytes
+		err = rows.Scan(&address, &pubkey)
+		if err != nil {
+			log.Error("Migrating accounts: failed to scan records", "err", err.Error())
+			return err
+		}
+		pk, err := crypto.UnmarshalPubkey(pubkey)
+		if err != nil {
+			log.Error("Migrating accounts: failed to unmarshal pubkey", "err", err.Error(), "pubkey", string(pubkey))
+			return err
+		}
+		pkBytes := sha256.Sum256(crypto.FromECDSAPub(pk))
+		keyUIDHex := hexutil.Encode(pkBytes[:])
+		_, err = sqlTx.Exec(`UPDATE accounts SET key_uid = ? WHERE address = ?`, keyUIDHex, address)
+		if err != nil {
+			log.Error("Migrating accounts: failed to update key_uid for imported accounts", "err", err.Error())
+			return err
+		}
+	}
```

Comment:
This section of code is not executed in your tests (at least that is what code climate reports) which suggest that the test is not retrieving any accounts from the DB.